data/Landouda.json

```
@@ -0,0 +1,13 @@
+{
+    "name": "Landouda",
+    "displayStatsPublic" : true,    
+    "type": "personal",
+    "bio": " I am a software engineering student",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/Landouda",
+        "icon": "FaGithub"
+      }
+    ]
+  }
```

Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

---

I think when merged, the GitHub Action will fix this